### DISCUSSION TITLE
Request to Display Individual Phase Power Values (L1, L2, L3) in Addition to Total Power

### DISCUSSION THREAD

**Discussion by robbieffm** [hoylabs/OpenDTU-OnBattery#1200] [Ideas]
Hi.

I wanted to ask if it would be possible to add a feature that displays the individual phase power values (L1, L2, L3) alongside the total power in the "Leistung / Netz xxx W" section. This would provide users with more detailed information, especially when their meter settings support retrieving values for individual phases.

Would this enhancement be feasible?

Regards
Robbie

![image](https://github.com/user-attachments/assets/7e321c60-8463-406d-b0fc-d78c85b1cc05)

**Comment by spcqike** [comment]
@schlimmchen 

ich hab grad diesen Request wieder gefunden und überlege, ob es jetzt, wo wir anfangen mehrere wechselrichter dynamisch zu steuern, nicht auch die Phase berücksichtigen können und wollen? 

In Deutschland ist es eigentlich egal, da der Zähler saldiert. Doch das ist nicht überall so (Österreich?) wenn ich mich recht erinnere ist das auch ein Grund warum das powermeter die Phasen kennt. Einige Leute haben daher pro Phase eine DTU. 

Macht es also Sinn, dem inverter im DPL die Phase mitzugeben und diese in der Entscheidung, welcher wechselrichter wie viel produzieren soll, zu berücksichtigen? 

Zusätzlich zu dem Request hier, die powermeter Leistung pro Phase darzustellen, wäre es dann auch sinnig, die wechselrichter Leistung pro Phase zu zeigen. 

Eventuell als übergeordneter Switch irgendwo a la „differenziere Phasen - ja / nein“. So dass entweder die Summen dargestellt und betrachtet werden oder eben primär die Phasen einzeln?

**Reply by Andrix82** [reply]
> In Deutschland ist es eigentlich egal, da der Zähler saldiert. Doch das ist nicht überall so (Österreich?)

Auch in Österreich saldieren die smartmeter problemlos

ich würde dies daher nur optional anzeigbar machen zwecks übersicht und ressourcen, für alle die kein interesse an möglichst hoher Netzneutralität haben und einfach nur wollen dass der zähler möglichst wenig hochzählt :) sind die werte irrelevant

**Comment by schlimmchen** [comment]
@spcqike Du vermischst hier zwar Themen, aber du hast mit allem recht.

Der Schalter "zeige Leistung der einzelnen Werte" gehört in die zu schaffende "Einstellungen" -> "Live View" Seite. Da kommen wir eher nicht mehr drumherum. Ich würde da auch gerne einen Schalter umlegen, nämlich "Zeige Inverter und Laderegler Allzeit-Total Energie" auf off...

Für den DPL gibt es da vermutlich nur wenig aufwändiges zu tun. Die Inverter müssen dann eine Ausprägung "Gruppe" erhalten, und jede Gruppe muss einem Ziel-Einspeisewert zugeordnet werden, z.B. einem PowerMeter Leistungswert, statt dem summierten Leistungswert. Dann ist lediglich noch das der Ausprägung zu filtern und der updateInverters() Funktion der passende Ziel-Leistungswert mitzugeben. Ist ein bisschen verkürzt, aber im Grunde ist es das.

In jedem Fall ist das aber ein Enhancement *nachdem* der Support für mehrere Inverter überhaupt mal hinreichend gut funktioniert.

Mach von mir aus ein Issue zu diesem Thema.

**Reply by Manos1966** [reply]
>Support für mehrere Inverter überhaupt mal hinreichend gut funktioniert.

❤️ ❤️ ❤️ ❤️

**Comment by schlimmchen** [comment]
Siehe #1354.
